guard/src/commands/test.rs

```diff
@@ -137,6 +138,9 @@ fn test_with_data(test_data_files: &[PathBuf], rules: &RulesFile<'_>, verbose: b
                     rules.evaluate(&root, &stacker)?;
                     let expectations = each.expectations.rules;
                     let stack = stacker.stack();
+                    if !each.name.is_none() {
+                        println!("Test Case: {}", each.name.unwrap());
+                    }
                     for each in &stack[0].children {
```

Comment:
Can we augment the output formatting for better alignment?

```
Test Case: <Test-Name>
  FAILED Rules:
     <rule_name>: Expected = FAILED, Evaluated = PASS
  PASS Rules:
     <rule_name>: Expected = PASS, Evaluated = PASS
  ```

Reply:
This makes sense from a readability standpoint